ra_aid/tools/programmer.py

```
@@ -26,16 +33,20 @@
 
     The programmer sees only what you provide, no conversation history.
 
-    Give detailed instructions but do not write their code.
+    Give detailed instructions including multi-file tasks but do not write their code.
 
-    They are intelligent and can edit multiple files.
+    The programmer cannot run commands.
 
     If new files are created, emit them after finishing.
 
     They can add/modify files, but not remove. Use run_shell_command to remove files. If referencing files you’ll delete, remove them after they finish.
```

Ruff (0.8.2)

Docstring contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF002)